.travis.yml

```diff
@@ -35,10 +33,11 @@ before_install:
   - composer self-update
   - if [ "$DEPS" = "dev" ]; then perl -pi -e 's/^}$/,"minimum-stability":"dev"}/' composer.json; fi
   - if [ "$SYMFONY_VERSION" != "" ]; then composer require symfony/symfony:${SYMFONY_VERSION} --no-update; fi
+  - phpunit --version
 
 install: composer update --prefer-dist $COMPOSER_FLAGS
 
-script: vendor/bin/simple-phpunit
```

Comment:
> or is that a problem that could come up again in the future?

Yep, e.g. when Symfony 4 will be released. PHPunit 5.6 requries symfony/yaml 3, so it'll fail to install or use an incompatible Yaml component version.